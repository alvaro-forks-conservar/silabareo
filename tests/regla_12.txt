árbol
arder
cargo
arpa
arte
arca
morfema
alergia
corcho
aguachirle
arma
arnés
neodarwinismo
marxismo
alba
aldea
algo
alpaca
alto
talco
alfa
alzar
aljibe
colcha
alrededor
alma
alna
andar
contar
banco
confesar
balanza
ángel
concha
enlazar
honrado
innegable