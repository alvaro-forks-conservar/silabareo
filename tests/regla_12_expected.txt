ár-bol
ar-der
car-go
ar-pa
ar-te
ar-ca
mor-fe-ma
a-ler-gia
cor-cho
a-gua-chir-le
ar-ma
ar-nés
ne-o-dar-wi-nis-mo
mar-xis-mo
al-ba
al-de-a
al-go
al-pa-ca
al-to
tal-co
al-fa
al-zar
al-ji-be
col-cha
al-re-de-dor
al-ma
al-na
an-dar
con-tar
ban-co
con-fe-sar
ba-lan-za
án-gel
con-cha
en-la-zar
hon-ra-do
in-ne-ga-ble